org.testeditor.web.dropwizard.xtext/src/test/java/org/testeditor/web/xtext/index/changes/ChainableChangeDetectorTest.xtend

```diff
@@ -0,0 +1,96 @@
+package org.testeditor.web.xtext.index.changes
```

Comment:
test is probably not necessary any more, if we choose minimalistic chaining through streaming api